tests/test_units/test_dataio.py

```diff
@@ -532,3 +530,28 @@ def test_norwegian_letters(encoding, mode, tmp_path):
 
     with open(tmp_path / "no-letters.yml", encoding=encoding) as f:
         assert yaml.safe_load(f) == {"æøå": "æøå"}
+
+
+@pytest.mark.parametrize(
+    "content",
+    [
+        field
+        for field, fieldinfo in AllowedContent.model_fields.items()
+        if fieldinfo.annotation is not type(None)
```

Comment:
This might be a bit flaky if we start to alter types on `AllowedContent`. IMO fine for now.